src/nativescript/user/user.ios.js

```diff
@@ -0,0 +1,24 @@
+import { User as CoreUser } from '../../core/user';
+
+export class User extends CoreUser {
+  static handleMICRedirectURL(redirectUri, micRedirectURL) {
```

Comment:
Who calls this method? I don't see a reference to this method name in the current master. I don't see any invocations in this PR either. What am I missing?

Reply:
This is no longer needed. I will remove it.